code/go/0chain.net/blobbercore/datastore/postgres_migration.go

```diff
@@ -0,0 +1,60 @@
+package datastore
+
+import (
+	"strings"
+	"time"
+
+	"gorm.io/gorm"
+)
+
+const (
+	TableNameMigration = "migrations"
+)
+
+// Migration migration history
+type Migration struct {
+	// Version format is "[table].[index].[column]"
+	//  + increase table version if any table is added or deleted
+	//  + increase index version if any index is changed
+	//  + increase column version if any column/constraint is changed
+	Version   string    `gorm:"column:version;primary_key"`
+	CreatedAt time.Time `gorm:"column:created_at"`
+	Scripts   []string  `gorm:"-"`
+}
+
+// After check if migrate is newer than latest
+func (m *Migration) After(latest *Migration) bool {
+	currentVersions := strings.Split(m.Version, ".")
+	latestVersions := strings.Split(latest.Version, ".")
+
+	for i := 0; i < 3; i++ {
+		if currentVersions[i] > latestVersions[i] {
```

Comment:
string comparisons are a bit tricker.
probably you'll have to check int comparison of these

Reply:
what if currentVersion is 2.13.0 and latestVersion is 3.12.0
this will return true. but should it be true?